evm/vm/forks/sharding/contracts/validator_manager.v.py

```diff
@@ -1,20 +1,20 @@
 # Events
 CollationAdded: __log__({
-    shard_id: indexed(num),
-    expected_period_number: num,
+    shard_id: indexed(int128),
```

Comment:
Sorry for that I haven't followed Vyper for a while, any reason that you're using signed integer 128 bit `int128` instead of `uint256` for this and following cases?

Reply:
Basically it's because `num` is renamed to `int128` and `num256` to `uint256`, but I'm fine with both. One advantage of going with `uint256` I can see is that converting `bytes32` into `int128` requires a few extra steps. 

first convert `bytes32` to `uint256`
(extra step)-> then mod 2^127
(extra step)-> then convert to `int128`

Reply:
I think it's reasonable to use `uint256` for the numbers that we already know that won't be negative.

Reply:
After I tried it out, it seems that the default type is `int128`(literal numbers and operators) . So if we turn all variables of type `int128` into type `uint256`, we will first need to convert literal number to `uint256` and use uint256-ish operators like `uint256_add`, `uint256_mul`, etc.

default:
```python
score: int128 = parent_score + 1
```

if set all variables to 'uint256' ==>

```python
score: uint256 =uint256_add(parent_score, convert(1, 'uint256'))
assert score > convert(1, 'uint256')
```

Reply:
Sorry I didn't realize that in spec `uint256` is used. I will update them to `uint256`.